server/src/main/java/io/crate/execution/engine/aggregation/impl/StandardDeviationAggregation.java

```diff
@@ -151,6 +104,10 @@ public StandardDeviationAggregation(Signature signature, BoundSignature boundSig
         this.boundSignature = boundSignature;
     }
 
+    protected abstract StdDevStateType<V> stdDevStateTypeInstance();
+
+    protected abstract V newVariance();
```

Comment:
This looks unused?

Reply:
It's used in the `getDocValueAggregator` on the parent function and since `V` is a generic we need to return a concrete new instance.